lib/twingly/url.rb

```diff
@@ -101,7 +99,7 @@ def normalized
       normalized_url.host   = normalized_host
       normalized_url.path   = normalized_path
 
-      self.class.internal_parse(normalized_url)
+      self.class.parse(normalized_url)
```

Comment:
This will convert an addressable uri to a string and then back to a addressable uri. I think we want to avoid that.